### PR TITLE
KAA-1516: Docker deployment - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Administration-guide/System-installation/Docker-deployment/index.md [hunk 1]
```diff
@@ -10,221 +10,182 @@ sort_idx: 40
...
+	{:.important}
+
+2. Install [Docker Сompose](https://docs.docker.com/compose/install/).
+
+3. Download Kaa debian package from the [official site](http://www.kaaproject.org/download-kaa/) or build your Kaa project locally (use `kaa-node.deb` located in `server/node/target/`).
+Put the `kaa-node.deb` file in the `server/containers/docker/` directory.
+
+4. Run the following command from the `server/containers/docker` directory.
+
+		docker build --build-arg setupfile=kaa-node.deb -t kaa-node:0.10.0
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Docker-deployment/index.md]: missing "." at the end of the command

File: doc/Administration-guide/System-installation/Docker-deployment/index.md [hunk 2]
```diff
@@ -10,221 +10,182 @@ sort_idx: 40
...
-    
-2. Download debian package from [official site](http://www.kaaproject.org/download-kaa/)
- or build your Kaa project locally (`kaa-node.deb` located in `server/node/target/`).
- 
-    Put `kaa-node.deb` into `server/containers/docker/` folder.
- 
-3. From `server/containers/docker` folder execute:
+1. Install [Docker Engine](https://docs.docker.com/engine/installation/).
+
+	If you use Linux, you can install the Docker Engine by running the [installation script](https://docs.docker.com/engine/installation/).
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Docker-deployment/index.md]: Incorrect link to the script. "https://get.docker.com/" should be used.

File: doc/Administration-guide/System-installation/Docker-deployment/index.md [hunk 3]
```diff
@@ -10,221 +10,182 @@ sort_idx: 40
...
+
+### Cluster node installation
+
+To install a cluster node:
+
+1. Run the following command from the `server/containers/docker/using-compose` directory.
+
+		python launch-kaa.py SQL-NoSQL
+
+2. Specify SQL-NoSQL databases which you wan to use.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Docker-deployment/index.md]: want

Comment (Polinaa) [inline doc/Administration-guide/System-installation/Docker-deployment/index.md]: This step refers to the first

File: doc/Administration-guide/System-installation/Docker-deployment/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Docker-deployment/index.md]: Fixed.

File: doc/Administration-guide/System-installation/Docker-deployment/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Docker-deployment/index.md]: Fixed.

File: doc/Administration-guide/System-installation/Docker-deployment/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Docker-deployment/index.md]: Fixed.

Comment (hunch7) [reply doc/Administration-guide/System-installation/Docker-deployment/index.md]: Fixed.

File: doc/Administration-guide/System-installation/Docker-deployment/index.md [hunk 4]
```diff
@@ -10,221 +10,183 @@ sort_idx: 40
...
+	{:.important}
+
+2. Install [Docker Сompose](https://docs.docker.com/compose/install/).
+
+3. Download Kaa debian package from the [official site](http://www.kaaproject.org/download-kaa/) or build your Kaa project locally (use `kaa-node.deb` located in `server/node/target/`).
+Put the `kaa-node.deb` file in the `server/containers/docker/` directory.
+
+4. Run the following command from the `server/containers/docker` directory.
+
+		docker build --build-arg setupfile=kaa-node.deb -t kaa-node:0.10.0.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/Docker-deployment/index.md]: Add space before dot "kaa-node:0.10.0 ."

Comment (hunch7) [reply doc/Administration-guide/System-installation/Docker-deployment/index.md]: Fixed.

**Comment by Polinaa** [comment]
LGTM

**Review by vadimol — APPROVED** [review]

**Pull request merged**